apps/files/src/FileUpload.js

```diff
@@ -8,7 +8,7 @@ class FileUpload {
 
   upload (file, additionalData = {}) {
     let xhr = new XMLHttpRequest()
-    xhr.responseType = 'json'
+    xhr.responseType = 'text'
```

Comment:
is "text" for the case where the body is empty ?

Reply:
by setting 'text' the response body holds the xml response as plain text - we can later try to parse and understand it.

in case you set 'json' the response is empty because the body has wrong format

Reply:
ah, of course... it's XML and not JSON